test/OrchardCore.Tests/Apis/Lucene/LuceneQueryTests.cs

```diff
@@ -0,0 +1,132 @@
+                titleBoost = string.Empty;
+                bodyBoost = "^2";
+
+                string bodyBoostQuery = baseQuery.Replace("{0}", titleBoost).Replace("{1}", bodyBoost);
+                content = await context.Client.GetAsync($"api/lucene/content?indexName={index}&query={bodyBoostQuery}");
+                queryResults = await content.Content.ReadAsAsync<LuceneQueryResults>();
+
+                contentItems = queryResults.Items.Select(x => ((JObject)x).ToObject<ContentItem>());
+
+                Assert.False(contentItems.All(x => x.DisplayText.Contains("Orchard", StringComparison.OrdinalIgnoreCase)));
+            }
+        }
+
+        [Fact]
+        public async Task SimpleQueryWildcardHasResults()
+        {
+            using (var context = new LuceneContext())
+            {
+                await context.InitializeAsync();
+                // Act
+                // Should find articles with "Orchard" in the title
+                string index = "ArticleIndex";
+                dynamic dynamicQuery = new
+                {
+                    from = 0,
+                    size = 1,
+                    query = new
+                    {
+                        simple_query_string = new
+                        {
+                            analyze_wildcard = true,
+                            fields = new string[] { "Content.ContentItem.DisplayText.Normalized" },
+                            query = "orch*"
+                        }
+                    }
+                };
+
+                string query = JsonConvert.SerializeObject(dynamicQuery);
+
+                var content = await context.Client.GetAsync($"api/lucene/content?indexName={index}&query={query}");
+                var queryResults = await content.Content.ReadAsAsync<LuceneQueryResults>();
+
+                Assert.NotEmpty(queryResults.Items);
+                var contentItems = queryResults.Items.Select(x => ((JObject)x).ToObject<ContentItem>());
+
+                Assert.Contains("Orchard", contentItems.First().DisplayText, StringComparison.OrdinalIgnoreCase);
+            }
+        }
+
+        [Fact]
+        public async Task TwoWildcardQueriesWithBoostHasResults()
+        {
+            using (var context = new LuceneContext())
+            {
+                await context.InitializeAsync();
+
+                // Act
+                // Should find articles with "Orchard" in the title
+                string index = "ArticleIndex";
+                string query =
+                    "{ \"from\": 0, \"size\": 10, \"query\":" +
+                        "{ \"bool\": { \"should\": [ " +
+                            "{ \"wildcard\": {  \"Content.ContentItem.DisplayText.Normalized\": { \"value\": \"orch*\", \"boost\": 2 } } }," +
+                            "{ \"wildcard\": { \"HtmlBodyPart\": { \"value\": \"orchar*\", \"boost\": 5 } } }" +
+                        "] } } }";
```

Comment:
here looks good, maybe helpful to just have the string unescaped in as a comment?